ern-api-gen/src/normalizeConfig.js

```diff
@@ -124,11 +126,15 @@ export default function normalizeConfig ({
   if (apiSchemaPath) {
     config.apiSchemaPath = apiSchemaPath
   }
-  if (!artifactId) {
-    config.artifactId = config.moduleName
+  if (!config.artifactId) {
+    config.artifactId = `react-native-${simpleName}-api`
```

Comment:
Not sure what `artifactId` is, so just double checking if that's ok for it to have the old naming convention.

Reply:
@belemaire I couldn't find any reference to artifactId usage , except in internal swagger `js` file.